contents/blog/why-we-raised-a-15m-series-b-ahead-of-schedule.md

```diff
@@ -0,0 +1,27 @@
+---
+date: 2021-06-14
+title: Why we raised a $15m Series B ahead of schedule
+author: joe-martin
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+By: [Joe Martin](https://www.linkedin.com/in/joemartinwords/), Product Marketing
+
+Last week we proudly announced we’d successfully raised $15 million in a Series B funding round, with support from Y Combinator Continuity Fund and GV (formerly Google Ventures). You may have seen [the news on VentureBeat](https://venturebeat.com/2021/06/10/open-source-product-analytics-platform-posthog-raises-15m/).
+
+What made this funding round different from most is that we hadn’t actually planned to raise a Series B yet. So, why did we?
+
+Over the last 12 months we’ve made huge improvements to our platform, including [introducing ClickHouse](https://posthog.com/blog/the-posthog-array-1-15-0) and [deciding to give away 100x more free events on PostHog Cloud](https://posthog.com/blog/the-posthog-array-1-25-0). This has led to a sustained surge in interest, to the point that we’ve been literally unable to keep up with demand. This is even despite doubling our headcount since the start of the year. 
+
+In some ways this is a nice position to be in. However, when we started to receive interest from investors again we saw an opportunity to address this bottleneck. We didn’t strictly need the money to continue as we were, but we realised raising funds ahead of schedule would enable us to grow quicker, build faster and to better support our growing community.
+
+In short, we had an opportunity to accelerate, so we decided to take it. 
+
+You may be curious what we’ll spend the money on. Our aim is to double the size of the team to 47 people by the end of the year, then double again in 2022. Right now we’re recruiting for Full Stack Engineers and Front End Developers, but we remain open to speculative applications too. [If you’re interested, get in touch!](https://posthog.com/careers)
+
+Other plans for the future include enhancing connectivity with data warehouses and merging our open source product into PostHog Free. It will take a few months to get there, but we ultimately want PostHog Free to be suitable for everyone from individual hobbyists to enterprise teams. 
```

Comment:
I'd probably describe this as the other way around - this would scare me that PostHog OS will flip into a non MIT licensed thing.

ie cool things in PostHog Free will go into PostHog Open Source :)

Reply:
Ah, good clarifiication. Committed a change.